processing/src/main/java/io/druid/query/aggregation/LongMaxAggregatorFactory.java

```diff
@@ -39,7 +39,7 @@
 
 /**
  */
-public class LongMaxAggregatorFactory extends AggregatorFactory
+public class LongMaxAggregatorFactory extends NullableAggregatorFactory
```

Comment:
Sure, This can be done, but i would prefer to do it after this PR is merged. I was even exploring more refactorings that can be done to merge the implementations and minimize the duplications.

Reply:
https://github.com/druid-io/druid/issues/5860

Reply:
done.